<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>60</y>
       <width>61</width>
       <height>501</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/Green Illustrated Earth Day Poster (3) (1).png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 244, 208);
alternate-background-color: rgb(252, 244, 208);
color: rgb(87, 125, 54);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(252, 244, 208);
color: rgb(87, 125, 54);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Mot de passe :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 244, 208);
alternate-background-color: rgb(252, 244, 208);
color: rgb(87, 125, 54);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Mot de passe oubli√© </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>230</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tab_employe">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>100</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>-5</y>
       <width>971</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../mariem/Atelier_Connexion_/Atelier_Connexion/mariem.qrc">:/mariem/logine1_page-0001.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(252, 244, 208);
color: rgb(87, 125, 54);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../mariem/Atelier_Connexion_/Atelier_Connexion/mariem.qrc">:/mariem/logo_login-removebg-preview (1).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>tab_employe</zorder>
    <zorder>label</zorder>
    <zorder>label_18</zorder>
    <zorder>pushButton</zorder>
    <zorder>label_14</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>lineEdit_username</zorder>
    <zorder>lineEdit_password</zorder>
    <zorder>label_13</zorder>
    <zorder>label_2</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../mariem/Atelier_Connexion_/Atelier_Connexion/mariem.qrc"/>
 </resources>
 <connections/>
</ui>
